### ISSUE TITLE
[Mod Compat] <Environmental, Atmospheric, Upgrade Aquatic, Quark, Traverse, Terrestria>

### ISSUE THREAD

**Issue by Roadhog360** [xanthian/variantbarrels#15]
https://www.curseforge.com/minecraft/mc-mods/environmental
https://www.curseforge.com/minecraft/mc-mods/atmospheric
https://www.curseforge.com/minecraft/mc-mods/upgrade-aquatic
https://www.curseforge.com/minecraft/mc-mods/quark
https://modrinth.com/mod/traverse
https://modrinth.com/mod/terrestria

While the first 4 are indeed Forge mods while this is a Fabric mod, I have had no issue running your barrels mod under Sinytra connector, and thus think these should be added in, since people on Forge are able to use this mod through Sinytra connector.

**Comment by xanthian** [comment]
I won't be adding any Forge specific mod compat i'm afraid, at least not with the Fabric release version.

IF I ever do a Forge specific release, I will consider this.

**Comment by Roadhog360** [comment]
Fabric mods can detect if a Forge mod id is loaded or if blocks from Forge mods are present, in pretty much the same way they'd do for Fabric mods so I don't see what the obstacle would be. Connector Extras helps to ensure this as well by bridging API components.

**Comment by xanthian** [comment]
Obstacle is time.

**Comment by Roadhog360** [comment]
True. Thought it was technical reasons. Take your time. Might PR this myself later. As well as tweak the barrel textures to be more unique and stuff.
